config/assisted-service/assisted-service-configmap.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+data:
+  ADMIN_USERS: ercohen,lgamliel,mfilanov,ygoldber
+  AGENT_DOCKER_IMAGE: quay.io/ocpmetal/assisted-installer-agent:latest
+  BASE_DNS_DOMAINS: ''
+  CHECK_CLUSTER_VERSION: 'False'
+  CONTROLLER_IMAGE: quay.io/ocpmetal/assisted-installer-controller:latest
+  CREATE_S3_BUCKET: 'true'
+  ENABLE_AUTH: 'False'
+  ENABLE_KUBE_API: 'true'
+  INSTALLER_IMAGE: quay.io/ocpmetal/assisted-installer:latest
+  JWKS_URL: https://api.openshift.com/.well-known/jwks.json
+  NAMESPACE: assisted-installer
+  OCM_BASE_URL: https://api-integration.6943.hive-integration.openshiftapps.com
+  OPENSHIFT_VERSIONS: '{"4.6":{"display_name":"4.6.12","release_image":"quay.io/openshift-release-dev/ocp-release:4.6.12-x86_64","rhcos_image":"https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.6/4.6.8/rhcos-4.6.8-x86_64-live.x86_64.iso","rhcos_version":"46.82.202012051820-0","support_level":"production"},"4.7":{"display_name":"4.7-pre-release","release_image":"quay.io/openshift-release-dev/ocp-release:4.7.0-rc.0-x86_64","rhcos_image":"https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/pre-release/4.7.0-fc.4/rhcos-4.7.0-fc.4-x86_64-live.x86_64.iso","rhcos_version":"47.83.202101171239-0","support_level":"beta"}}'
```

Comment:
Is there any way we can avoid adding another place we store this kind of information?

Reply:
Please add an option to generate this out of https://github.com/openshift/assisted-service/blob/7d53cb0a8ece947fc63d15756ea2b0eb787a7370/default_ocp_versions.json

Also please verify that this generation gets done in https://github.com/eranco74/assisted-installer-deployment/blob/master/tools/new_minor_OCP_version.py

Reply:
I'm going to remove most of the yaml in config/assisted-service. They are just copies of the manifests generated into the build/assisted-installer directory. So far I haven't discovered a need to archive them into the repo, so I'll add them .gitignore.

Reply:
File removed.